website/docs/d/cm_server_template.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "rightscale"
+page_title: "Rightscale: server_template"
+sidebar_current: "docs-rightscale-datasource-server_template"
+description: |-
+  Defines a server template datasource to operate against.
+---
+
+# rightscale_server_template
+
+Use this data source to get the ID or other attributes of an existing server template for use in other resources.
```

Comment:
Let's specify that its a template that exists IN your account.  It does not talk to the library at all to find any incarnations that match - it only filters with what exists in your account.

Reply:
Updated